CHANGES.txt

```diff
@@ -24,6 +24,7 @@
  * Add diagnostic events for guardrails (CASSANDRA-17197)
  * Pre hashed passwords in CQL (CASSANDRA-17334)
  * Increase cqlsh version (CASSANDRA-17432)
+ * Added cqlsh custom auth provider support (CASSANDRA-16456)
```

Comment:
@bhouse99 remove this line, I ll take care of that on the merge itself. No worries. We would have to rebase / fix it all over again as other people are adding stuff there in the meanwhile.

Reply:
Kk, will do will avoid modifying in the future.